### PR TITLE
Refactor: lazy-load governance widget

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

Part of #3323

The governance widget makes a few requests to CGW from below the fold. This PR defers the rendering of this widget until the user scrolls to it.

### CODE REVIEW

File: src/components/dashboard/GovernanceSection/GovernanceSection.tsx [hunk 1]
```diff
@@ -85,73 +82,76 @@ const GovernanceSection = () => {
...
+    window.addEventListener('scroll', handleScroll)
+    return () => window.removeEventListener('scroll', handleScroll)
   }, [])
 
-  if (!isLayoutReady) {
-    return null
-  }
+  return (
+    <>
+      {hasScrolled && <InfiniteScroll onLoadMore={onVisible} />}
```
Comment (compojoom) [inline src/components/dashboard/GovernanceSection/GovernanceSection.tsx:134]: Why do you need hasScrolled? Can't we just render the infinitescroll all the time?

Comment (katspaugh) [reply src/components/dashboard/GovernanceSection/GovernanceSection.tsx:134]: The page is small in the beginning and this block is always in the viewport initially.

**Review by compojoom — APPROVED** [review]
👍

**Comment by francovenica** [comment]
LGTM. 
The widget doesn't load anymore as soon as you enter in the dashboard (since the widget is visible for a second), instead it loads as you scroll down.

**Pull request merged**